src/index.html

```diff
@@ -1,18 +1,73 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
```

Comment:
In general use the page class as the body, and then add page__newblock (as a Mix Block) for the direct descendants of the body tag